apps/sms/services/js/drafts.js

```diff
@@ -138,6 +149,13 @@
     },
 
     /**
+     * Returns list of all thread less drafts.
+     */
+    getAllThreadless: function () {
+      return draftIndex.get(null) || [];
```

Comment:
mmm it shouldn't, `Drafts.request` is doing `draftIndex = new Map(records || []);` ?

Where was it failing ? in B2G Desktop ?